### PR TITLE
contribution PR

### PR DESCRIPTION
**Pull request by chaos-ui**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by chaos-ui** [comment]
add needs help

**Review by chaos-ui — COMMENTED** [review]
badge

File: CodeFactor Badge [hunk 1]
```diff
@@ -0,0 +1 @@
+[![CodeFactor](https://www.codefactor.io/repository/github/chaos-ui/active-directory-b2c-javascript-nodejs-webapi/badge)](https://www.codefactor.io/repository/github/chaos-ui/active-directory-b2c-javascript-nodejs-webapi)
```
Comment (derisen) [inline CodeFactor Badge:1]: I believe you accidently checked this in -can you make sure?

**Review by derisen — COMMENTED** [review]
Thanks for this @chaos-ui ! Can you please address comments?

**Pull request merged**